### ISSUE TITLE
OpenSSL information on errors with requests that are http based

### ISSUE THREAD

**Issue by andy-maier** [pywbem/pywbem#2049]
With the "requests" based communication (PR #2023), we are getting OpenSSL information on errors with requests that are http based and should not involve SSL.
```
(pywbem3) C:\Users\kscho\dev\pywbem\examples>enuminstances.py
Usage: C:\Users\kscho\dev\pywbem\examples\enuminstances.py server_url username password namespace' ' classname
Using internal defaults
Requesting url=http://localhost, ns=root/cimv2, class=CIM_ComputerSystem
TARGET_URL http://localhost:5988/cimom
Operation failed: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it; OpenSSL version used: OpenSSL 1.0.2o 27 Mar 2018
```

Initial analysis: The "OpenSSL version used ..." text is inserted by pywbem.

**Discussion point:** This could be improved by only inserting it for scheme='https'.

**Comment by andy-maier** [comment]
We agreed to address this as proposed.

**Comment by andy-maier** [comment]
Fixed in PR #2023.
